packages/dcos-integration-test/extra/conftest.py

```diff
@@ -34,13 +34,11 @@ def pytest_collection_modifyitems(session, config, items):
 @pytest.fixture
 def vip_apps(dcos_api_session):
     vip1 = '6.6.6.1:6661'
-    test_app1, _ = get_test_app()
-    test_app1['portDefinitions'][0]['labels'] = {
-        'VIP_0': vip1}
-    test_app2, _ = get_test_app()
-    test_app2['portDefinitions'][0]['labels'] = {
-        'VIP_0': 'foobarbaz:5432'}
-    vip2 = 'foobarbaz.marathon.l4lb.thisdcos.directory:5432'
+    test_app1, _ = get_test_app(vip=vip1)
+    name = 'foobarbaz'
```

Comment:
That is more confusing as there is a test_vip and this fixture is not used there. Rather, this fixture is used for resiliency/test_aws_cf.py. This isn't being used generically, but nonetheless let's compromise with `myvipapp`